### PR TITLE
Update solutions-engineer.md

### PR DESCRIPTION
**Pull request by misterEggroll**

### CODE REVIEW

File: engineering/solutions-engineer.md [hunk 1]
```diff
@@ -109,40 +59,10 @@ Work you will be doing:
...
-
-If you're interested in shaking hands and joining the rocketship, please reach out to one of our team members mentioned here. If you've received an email or note from us, please respond accordingly and we will take care of it.
-
-- Abhishek (abhishek.ekka@skit.ai)
----
+- Understanding of Computer Science fundamentals, using them to effectively design and write code in python.
+- Hands-on experience developing microservices/projects in Python is preferrable.
+- Hands-on experience working with RDBMS like PostgreSQL.
+- Experience working with gRPC is a plus.
+- Experience working with container platforms like Docker, Kubernetes is a plus.
```
Comment (runprabhsharun) [inline engineering/solutions-engineer.md:67]: I think we can add 
1. Basics of cloud computing and cloud technologies like AWS, Azure is preferable 
2. Basics of any monitoring stack like Grafana, Prometheus, OTel, etc
3. SQL and SQL dbs

Comment (adarshmoni23) [reply engineering/solutions-engineer.md:67]: could also mention "Knowledge on Lambdas is a plus"(or we could mention that knowledge on AWS stack is preferable) .Other than that, at a general level, JD looks fine and has covered all aspects

Comment (misterEggroll) [reply engineering/solutions-engineer.md:67]: > could also mention "Knowledge on Lambdas is a plus"(or we could mention that knowledge on AWS stack is preferable) .Other than that, at a general level, JD looks fine and has covered all aspects

> Basics of cloud computing and cloud technologies like AWS, Azure is preferable

@runprabhsharun @adarshmoni23 
This is very specific. Considering 0-2 YOE is the range of experience we're looking for, it's a long shot to know this.
Have instead added: "_Experience with AWS or any comparable cloud technology is a plus._"

> SQL and SQL dbs

@runprabhsharun  This is mentioned under:
"_Hands-on experience working with RDBMS like PostgreSQL._"

Comment (adarshmoni23) [reply engineering/solutions-engineer.md:67]: @misterEggroll "Experience with AWS or any comparable cloud technology is a plus." -> yeah this works.

File: engineering/solutions-engineer.md [hunk 2]
```diff
@@ -2,103 +2,53 @@
...
-- You will be building ML models on our core ML platform; and monitor, diagnose and maintain active deployments.
-- You're expected to help build the Solutions stack whose goal it is to reduce the amount of effort put into building our voicebots. 🚀
-- You will be involved in analyzing client calls 🔍 where you're expected to use either data pipelines or SQL to get insights into caller behavior.
-
-### 📋 Technical Requirements
-
-**Mandatory Requirements:**
-
-- Professional experience of at least 1 year.
+- Professional experience of at 0-2 years.
```
Comment (joelmathew-vai) [inline engineering/solutions-engineer.md]: Professional experience of ~~at~~ 0-2 years.

**Pull request merged**